server/mailet/mailets/pom.xml

```diff
@@ -221,6 +221,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I am not sure if you ok with that?
Please tell me

Reply:
seems fine it does not drag any dependency